packages/components/src/internal/components/editable/EditableGrid.tsx

```diff
@@ -917,7 +919,12 @@ export class EditableGrid extends PureComponent<EditableGridProps, EditableGridS
                     width = metadata.minWidth;
                 }
             }
-
+            if (qCol.hasHelpTipData) {
+                if (!metadata) {
+                    metadata = {};
+                }
+                metadata.hideTitleTooltip = true;
```

Comment:
We probably don't want to be altering metadata here, as it comes from a prop, and would be an unexpected side effect

Reply:
Ah, good point.